packages/react/scroll-area/src/ScrollArea.tsx

```diff
@@ -725,6 +726,7 @@ const ScrollAreaScrollbarImpl = React.forwardRef<
             element.releasePointerCapture(event.pointerId);
           }
           document.body.style.webkitUserSelect = prevWebkitUserSelectRef.current;
+          if (context.viewport) context.viewport.style.scrollBehavior = '';
```

Comment:
Is there any risk that this event would fire? I'm curious as the capture release code is above this.

Reply:
I don't think so, what are you thinking?

Reply:
No, this looks good. Just raising the question just in case 😄